rl/algorithms/Algorithm.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC, abstractmethod
+
+from rl.algorithms.Config import Config
+
+
+class Algorithm(ABC):
+    def __init__(self, logger) -> None:
+        self.logger = logger
+        self.config = None
+
+    @abstractmethod
+    def get_reward(self, reward: float) -> float:

```

Comment:
Próbuję zrozumieć co ta funkcja robi.

Jak dobrze rozumiem ona przyjmuje nagrodę jako parametr, zapisuje go, a potem zwraca nagrodę?

Jeżeli tak to może lepszą nazwą byłoby "store_reward"? Lepiej by odzwierciedlało wówczas co ta funkcja robi.

Reply:
może byc store_reward, nie zwraca nagrody, ale gdzieś zapisuje. Nagroda jest otrzymywana za poprzedni ruch, więc nie można jej dać w make_action, a gdzieś trzeba